docs/deployment-arbitrary-cluster.md

```diff
@@ -0,0 +1,50 @@
+# Deployment on Arbitrary Cluster
+
+In this guide, we will cover the installation of etworkManager
+and kubernetes-nmstate on your arbitrary cluster.
+
+## Requirements
+
+This guide requires you to have your own Kubernetes/OpenShift cluster. If you
+don't have one and just want to try kubernetes-nmstate out, please refer to
+the [deployment on local cluster](deployment-local-cluster.md) guide.
+
+In order to get kubernetes-nmstate running, NetworkManager
+must be installed on the node.
+
+### NetworkManager
+
+kubernetes-nmstate containers communicate with a NetworkManager instance running
+on the node using D-Bus. Make sure that NetworkManager is installed and running
+on each node.
+
+```shell
+yum install NetworkManager
+systemctl start NetworkManager
+```
+
+## kubernetes-nmstate
+
+Finally, we can install kubernetes-nmstate on our cluster.
+
+### Kubernetes
+
+```shell
+# Install k8s resources and kubernets-nmstate operator
+for manifest in "service_account.yaml role.yaml role_binding.yaml crds/nmstate_v1_nodenetworkstate_crd.yaml operator.yaml"; do
+    kubectl apply -f https://raw.githubusercontent.com/nmstate/kubernetes-nmstate/master/deploy/0.0.3/$manifest
```

Comment:
Outdated link and I'm notr sure about manifests neither. Is this still fine?

Reply:
We have to decide on this we have three options:
- just tag master on versions
- branch master with version and tag
- just one track master with all the version inside deploy

Reply:
So as we talk about, for arbitrary clusters, we use https://github.com/nmstate/kubernetes-nmstate/releases and for local cluster deploy/ manifests changing operator.yaml to use quay.io